### PR TITLE
SnowFlake Integration for EvaDB

### PR DESCRIPTION
**Pull request by kslohith**
Please suggest if this feature needs more test cases

### CODE REVIEW

File: test/third_party_tests/test_native_executor.py [hunk 1]
```diff
@@ -228,6 +228,26 @@ def test_should_run_query_in_clickhouse(self):
...
+    def test_should_run_query_in_snowflake(self):
+        # Create database.
+        params = {
+            "user": "eva",
+            "password": "password",
+            "account": "account_number",
+            "database": "EVADB",
+            "schema": "SAMPLE_DATA",
+            "warehouse": "warehouse",
+        }
```
Comment (xzdandy) [inline test/third_party_tests/test_native_executor.py:240]: Is this a valid connection?

Comment (kslohith) [reply test/third_party_tests/test_native_executor.py:240]: No, it is not, do we need to test it against a real snowflake account ?

Comment (xzdandy) [reply test/third_party_tests/test_native_executor.py:240]: Yes. Otherwise the test will not run. Does snowflake has any free account we can test with?

Comment (xzdandy) [reply test/third_party_tests/test_native_executor.py:240]: I see. Then we can skip this test. Use `@pytest.mark.skip(reason="...")`

**Comment by xzdandy** [comment]
Please also update the documentation page: https://evadb.readthedocs.io/en/stable/source/reference/databases/index.html Thanks!

**Review by xzdandy — APPROVED** [review]

**Pull request merged**